### PR TITLE
fix(devices): Avoid reporting stale last-access times when feature is disabled.

### LINKED ISSUE #2142: isLastAccessTimeEnabledForUser() should disable reads as well as writes
We have a feature-flag `isLastAccessTimeEnabledForUser()` that's used to pref on/off the tracking of "last acitve" timestamps in redis.  However, it only controls the writing of those timestamps, so if you pref it on and then off again you can end up reading stale data:

* We pref `isLastAccessTimeEnabledForUser()` for some subset of users
* Time passes, those users sync, and their last-access timestamps are updated in redis
* We pref off `isLastAccessTimeEnabledForUser()` for some operational reasons
* Those users no longer get their last-access timestamps updated in redis, but the old values stick around untouched
* When those users go to their "devices and apps" view on the web, we read the old stale last-access timestamps out of redis and display times that are wildly out of date

I suspect this is what's causing https://bugzilla.mozilla.org/show_bug.cgi?id=1403275 and similar.  @jrgm did we pref this on and back off again at some point?

One solution here is to have the `isLastAccessTimeEnabledForUser()` also pref off the reading of these timestamps from redis, so that we don't read stale data.

Another solution would be to just purge the current contents of redis, but I don't know what other effects that might have.

### PR DESCRIPTION
**Pull request by rfk**
Hopefully fixes #2142, pushing for CI

### CODE REVIEW

**Comment by rfk** [comment]
Needs tests, but at least it doesn't cause any existing tests to fail...

**Comment by rfk** [comment]
Test added, @mozilla/fxa-devs r?

File: test/remote/db_tests.js [hunk 1]
```diff
@@ -210,7 +210,16 @@ describe('remote db', function() {
...
+          return db.sessions(account.uid)
+        })
+        .then(sessions => {
+          assert.equal(sessions.length, 1, 'sessions contains one item')
+          assert.equal(Object.keys(sessions[0]).length, 18, 'session has correct number of properties')
+          assert.equal(sessions[0].uid, account.uid, 'uid property is correct')
+          assert.equal(sessions[0].lastAccessTime, undefined, 'lastAccessTime not reported if disabled')
+
+          // Re-enable session token updates
           lastAccessTimeUpdates.enabled = true
```
Comment (philbooth) [inline test/remote/db_tests.js]: Should we add similar test coverage for `db.devices`?

**Comment by rfk** [comment]
> Probably worth adding the test for db.devices too though, I reckon.

You're right, thanks - I went looking for existing "lastAccessTimeUpdated" tests and modified them rather than thinking about which new codepaths need to be tested.  Will add.

**Comment by rfk** [comment]
Updated with a new test and related es6-ification; @philbooth mind a quick sanity-check before merge?

**Comment by philbooth** [comment]
> @philbooth mind a quick sanity-check before merge?

Perfect! 👍

**Pull request merged**